docs/index.rst

```diff
@@ -1,309 +1,2 @@
-.. pyuvdata documentation master file, created by
```

Comment:
Does this mean that we don't actually need this file to make the docs?

Reply:
We do not. I had initially excluded it from the repo entirely in the .gitignore, which made it much less likely that it could get screwed up. But @dannyjacobs didn't like that when you clicked on the "edit on github" button at the top of RTD it gave you a 404 error, so he made a stub file. I'd still prefer not to have it, every time I build the docs locally on my machine I have to be careful to make sure I don't commit the changes to that file, which is what the .gitignore is supposed to protect me against. And I clearly made a mistake a few months ago and committed changes that I shouldn't have.

Reply:
Can we add it to the `.gitignore` now that it's back to being a stub? I don't mind either way, we can also merge as is.

Reply:
That won't do anything because git is tracking it already. `.gitignore` only affects untracked files. The only way to really fix the problem is to remove it from the repo, but then we'll go back to having a 404 error if you click the "edit on github" link on RTD. @dannyjacobs do you still really care that that link doesn't 404? Because this stub only fixes that link on the main docs page, not on any of the subpages, which currently give a 404 error.

Reply:
apparently there is a way to remove that link: https://docs.readthedocs.io/en/stable/guides/remove-edit-buttons.html

Reply:
In that case, I'd vote for getting rid of this file (and adding it to the `.gitignore`), and the "edit on github" link. @dannyjacobs is that okay with you?

Reply:
Ok, done. It was already in the .gitignore. I have verified that the link is gone on RTD.